### PR TITLE
Add min_lr in lr_scheduler

### PR DESCRIPTION
**Pull request by ZhiyuanChen**
If the training step is too long, the warmup lr could be zero.

### CODE REVIEW

**Comment by andsteing** [comment]
I don't think lr would be zero : we start with `step=1`

https://github.com/google-research/vision_transformer/blob/ba9a85bdc430daf4da7b9da67b486a4e0f5bb278/vit_jax/train.py#L159-L161

and during the warmup period the lr only depends on the number of warmup steps, with a linear increase from `lr * 1/warmup_steps` to the full `lr`
